src/main/resources/META-INF/plugin.xml

```diff
@@ -169,6 +169,9 @@
 
         <!-- Structure view -->
         <lang.psiStructureViewFactory language="Robotframework" implementationClass="amailp.intellij.robot.extensions.RobotPsiStructureViewFactory"/>
+
+        <!-- Robot keyword reference -->
+        <psi.referenceContributor implementation="amailp.intellij.robot.psi.reference.KeywordReferenceContributor"/>
```

Comment:
Why is this needed? AFAIK the reference contributor serves for non standard references but references are usually retrieved through the getReference method as they currently are. See [here](https://github.com/JetBrains/intellij-community/blob/master/platform/core-api/src/com/intellij/psi/PsiReference.java#L27)